Gulpfile.js

```diff
@@ -187,7 +187,18 @@ function spawnTask(command, done, args)
         console.error("Failed to start subprocess.");
         done(err);
     });
-    lCP.on("exit", () => done());
+
+    // catch non-zero exit statii so that CI can understand when task fails
+    lCP.on("close", (code, signal) => {
```

Comment:
- What's `signal` doing?
 - Can `done` not take the code and figure out if it was error or not?

Reply:
`signal` seems to be `null` here (in the few cases I've tested), but it's part of my intellisense signature analysis, so I left it in for good measure. Done _can_ take the code, but I was reading the gulp docs and `done` is an error-first call back, so they suggest passing it an `Error`, if you're to pass it anything at all. When we don't, it's NBD, but the output is gross. With these few lines of code we can prettify things on the UI-side.

Reply:
Can you post the diff in output? Our error message does not actually convey any useful info, so the real difference is in `gulp`'s printing logic depending on code vs error. If it is really ugly then im okay with this but from a source point of view creating an unnecessary error is not as nice as just forwarding the code

Reply:
Can you remove `signal`? `eslint` would be complaining that it's unused

Reply:
done!